themes/jaeger-docs/layouts/index.redirects

```diff
@@ -0,0 +1,11 @@
+{{- $latest := .Site.Params.versions.latest }}
+{{- $docs := readDir "content/docs/1.5" }}
```

Comment:
why does this refer to 1.5 specifically?

Reply:
is it because the non-versioned URLs only existed in 1.5? (actually 1.6, the current ver).
would be good to add a comment about that

Reply:
Yeah, exactly. This ensures that all of the non-versioned docs are covered in the URL scheme.